.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,55 @@
+name: Build Wheels
+
+on: 
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.config.os }} - ${{matrix.python-build}}
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        config: 
+          - os: ubuntu-20.04
+            shell: bash
+          # - os: macos-10.15
+          #   shell: bash
+          - os: windows-2019
+            preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
+            shell: cmd
+          # These are the platform build strings provided to cibuildwheel, with wildcarding.
+          # See https://cibuildwheel.readthedocs.io/en/stable/options/#build-skip
+        python-build: ['cp39*64', 'cp310*64']
+    defaults:
+      run:
+        # Annoyingly required here since `matrix` isn't available in
+        # the `shell` property of individual steps.
+        shell: ${{ matrix.config.shell }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Bootstrap
+        uses: ./.github/bootstrap_platform
+
+      - name: Build wheels
+        run: |
+          ${{ matrix.config.preamble }}
+          pip install cibuildwheel
+          cibuildwheel --output-dir wheelhouse
+        env: 
+          # Windows + Mac runs use this environment variable, as they execute directly
+          # on the runner. Linux runs are somewhat different, and are containerised fully,
+          # therefore they get their toolchain path from pyproject.toml
+          CMAKE_TOOLCHAIN_FILE: ${{ github.workspace }}/.conan/conan_paths.cmake
+          CIBW_BUILD: ${{ matrix.python-build }}
+          CIBW_SKIP: '*musllinux* *arm64*'
+          PIP_VERBOSE: 1
```

Comment:
[question] Did we get to the bottom of the `manylinux201` vs `2014` question? (https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/.github/workflows/python-package.yml#L171)

Reply:
No we didn't, i've asked on the OTIO slack : https://academysoftwarefdn.slack.com/archives/CMQ9J4BQC/p1666361161870199

Reply:
Looks like CentOS 6 support was the main reason, they plan to eventually update. What do you think @foundrytom, how backwards compatible do you wanna be here?

Reply:
For now, compatibility with Centos 7 is important, as a significant portion of the industry is there.

Reply:
[fix] Could we just pin this though, to save us breaking compatibility at some random point when they update the defaults? (Though hopefully well end up with a CI check that pip installs inside of VFX platform containers).

Reply:
Yeah, i had a similar thought, I think that's sensible. Will do

Reply:
[dd10abd](https://github.com/OpenAssetIO/OpenAssetIO/pull/665/commits/dd10abd68a74b2ecf217001aa5d2261e0718be19)